.github/workflows/cross-version-tests.yml

```diff
@@ -125,11 +126,6 @@ jobs:
         with:
           python-version: ${{ steps.get-python-version.outputs.version }}
       - uses: ./.github/actions/setup-pyenv
-      - name: Remove unneeded libraries
-        run: |
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /usr/local/lib/android
```

Comment:
Freeing disk space after setting up python unsets the python set up by `actions/setup-pyhton`.